src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -158,7 +159,7 @@ enum class ConfigType : uint16_t {
  *
  * @return The corresponding string name.
  */
-inline const char* to_string(ConfigType type) {
+P1_CONSTEXPR_FUNC const char* to_string(ConfigType type) {
```

Comment:
Ideally, we'd also make the struct constructors P1_CONSTEXPR_FUNC as well. I've had issues since:
`explicit InterfaceID(TransportType type, uint8_t index = 0)` isn't a `constexpr`.

Reply:
Sure I can add that